### ISSUE TITLE
[v13.0.0-beta.0] Missing context menu items

### ISSUE THREAD

**Issue by microbit-robert** [RaspberryPiFoundation/blockly#9704]
### Check for duplicates

- [x] I have searched for similar issues before opening a new one.

### Description

The following context menu items are currently missing if the aim is parity with the keyboard navigation plugin:

- Move block
- Cut / Copy / Paste
- Edit block contents

In addition, the following context menu items are missing keyboard keys:

- Delete
- Duplicate (shortcut not implemented - see https://github.com/RaspberryPiFoundation/blockly/issues/9703)

### Reproduction steps

1. 
2. 
3.

### Stack trace

```shell

```

### Screenshots

https://raspberrypifoundation.github.io/blockly-keyboard-experimentation/screenreader/
<img width="214" height="359" alt="Image" src="https://github.com/user-attachments/assets/ac20b055-83f8-45b1-819a-8b4351add5bd" />

v13.0.0-beta.0 running locally
<img width="166" height="215" alt="Image" src="https://github.com/user-attachments/assets/1ebca5af-7410-4923-8e53-54e578b13dc0" />

**Comment by gonfunko** [comment]
I talked with the team and based on the decision in https://github.com/RaspberryPiFoundation/blockly-keyboard-experimentation/issues/466 we don't want move/edit block contents, and decided against adding cut/copy/paste on the basis that we haven't had them to date. Display of shortcut keys is being handled in https://github.com/RaspberryPiFoundation/blockly/pull/9785
